src/components/hv-picker-field/field-label/types.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import type { DOMString, StyleSheet } from 'hyperview/src/types';
+
+export type Props = {
+  focused: boolean;
+  labelFormat: string | null | undefined;
```

Comment:
Do we need the ' | null ' in all of these?

Reply:
I think these were generated based on the old flow style, which permitted null or undefined. We might want to revisit this / tighten up some of these types in the future.